### DISCUSSION TITLE
Will CoreForms also support Linux?

### DISCUSSION THREAD

**Discussion by simonegli8** [CoreForms/docs#3] [Ideas]
Hello,

I'm working for the SolidCP OpenSource webhosting control panel. The frontend is a huge WebForms code base, the rest of the application we successfully moved to NET Core with the aid of CoreWCF. We'd like to port the frontend also to NET Core, but our motivation is to have it run on Linux also, since we want to support Linux too. Do you have plans to support Linux too? Is CoreForms a port of the Microsoft Reference sources WebForms or of Mono WebForms? Are there many dependecies on Windows in the codebase? If you are interested, can I help to port CoreForms to Linux?

**Comment by MichaelKetting** [comment]
Hello @simonegli8,

Our implementation is aligned with the official Microsoft version, not based on Mono WebForms. There's a fair number of dependencies on Windows system calls in CoreForms. We know how to get rid of them, but it would require an actual business case. It would be great if you could contact us at  coreforms@rubicon.eu to further this discussion. 

Thank you for the offer of contribution. Right now, CoreForms isn't set up to support contributions (the source code isn't hosted on Github, for example) and there's also the whole copyright/legal stuff with a closed-source project so I'm not sure contributions will be possible. It is something that could be discussed via e-mail though.

~Michael
